src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html

```diff
@@ -2,73 +2,94 @@
+          <button class="btn filled-btn" (click)="changeStatus()" [disabled]="counter">{{ 'status.save' | translate }}</button>
+
+        </div>
+
       </div>
-      <mat-card class="candidate-skills">{{CANDIDATES_INFO.techSkills}}</mat-card>
-      <mat-card class="candidate-skills">additional skills</mat-card>
+      <mat-card class="candidate-skills">{{ CANDIDATES_INFO.candidateTechSkills || null }}</mat-card>
+      <mat-card class="candidate-skills">{{ CANDIDATES_INFO.additionalSkills || null }}</mat-card>
       <div class="raiting two-column">
-        <mat-card>{{CANDIDATES_INFO.email}}</mat-card>
-        <mat-card>{{CANDIDATES_INFO.skype}}</mat-card>
-        <mat-card>{{CANDIDATES_INFO.location}}</mat-card>
-        <mat-card>sertificates</mat-card>
-        <mat-card>{{CANDIDATES_INFO.phone}}</mat-card>
-        <mat-card>{{CANDIDATES_INFO.languages}}</mat-card>
-        <mat-card>current job</mat-card>
+        <mat-card>{{ CANDIDATES_INFO.email }}</mat-card>
+        <mat-card>{{ CANDIDATES_INFO.skype }}</mat-card>
+        <mat-card>{{ CANDIDATES_INFO.location }}</mat-card>
+        <mat-card>{{ CANDIDATES_INFO.professionaCertificates }}</mat-card>
+        <mat-card>{{ CANDIDATES_INFO.phone }}</mat-card>
+        <mat-card>{{ CANDIDATES_INFO.candidateLanguages }}</mat-card>
+        <mat-card>{{ currentJob }}</mat-card>
         <mat-card>time to contact</mat-card>
       </div>
     </div>
     <div class="section">
-      <mat-card class="user-name">{{ USERS_INFO.name }}{{ 'feedback' | translate }}</mat-card>
+      <mat-card class="user-name">{{ userAuthName }}{{ 'feedback' | translate }}</mat-card>
+
+      <div class="custom-slider" *ngIf="!isRecruiter">
 
-      <div class="custom-slider" *ngIf="USERS_INFO.role === 'admin'">
         <div class="custom-slider-slide">
           <mat-slider
             thumbLabel
             tickInterval="1"
             step="1"
             min="0"
-            max="100"
+            max="{{ grade }}"
             aria-label="units"
             (change)="onChangeRange($event)">
           </mat-slider>
         </div>
         <div class="custom-slider-info">
-          <span>{{ sliderValue }}</span>
+          <span >{{ sliderValue || 0 }}</span>
         </div>
       </div>
 
-      <mat-form-field class="textarea_container" appearance="fill"  *ngIf="USERS_INFO.role === 'recruiter'">
-        <mat-label>{{ USERS_INFO.name }} feedback</mat-label>
-        <textarea [cdkTextareaAutosize] class="textarea" matInput placeholder="Place your review here..."></textarea>
+      <mat-form-field class="textarea_container" appearance="fill">
+        <mat-label>{{ userAuthName }} feedback</mat-label>
+        <textarea [cdkTextareaAutosize] 
+          class="textarea" 
+          [(ngModel)]="userReview"
+          matInput placeholder="Place your review here..."
+          >
```

Comment:
`>` should either be at the end of the previous line (as it is everywhere in the project) or aligned with opening `<`